### ISSUE TITLE
django-rest-swagger 0.3.10 is not compatible with python 3.6

### ISSUE THREAD

**Issue by timur-orudzhov** [marcgibbons/django-rest-swagger#719]
I get this exception, when i use your app with python 3.6. My environment is:

    django-rest-swagger==0.3.10
    djangorestframework==3.3.3

Problem is `commonprefix` does not support set type anymore. This link proves it - https://bugs.python.org/issue28527. Unfortunately, I can't run my application on previous version of python. Do you have any plans to support version 0.3.* with latest python interpreter ? Thanks in advice for any information.

```
Traceback (most recent call last):
  File "/home/timur/venvs/freestone/lib/python3.6/site-packages/django/core/handlers/base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/timur/venvs/freestone/lib/python3.6/site-packages/django/core/handlers/base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/timur/venvs/freestone/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/timur/venvs/freestone/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/timur/venvs/freestone/lib/python3.6/site-packages/rest_framework/views.py", line 466, in dispatch
    response = self.handle_exception(exc)
  File "/home/timur/venvs/freestone/lib/python3.6/site-packages/rest_framework/views.py", line 463, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/timur/venvs/freestone/lib/python3.6/site-packages/rest_framework_swagger/views.py", line 116, in get
    apis = [{'path': '/' + path} for path in self.get_resources()]
  File "/home/timur/venvs/freestone/lib/python3.6/site-packages/rest_framework_swagger/views.py", line 151, in get_resources
    resources = urlparser.get_top_level_apis(authorized_apis_list)
  File "/home/timur/venvs/freestone/lib/python3.6/site-packages/rest_framework_swagger/urlparser.py", line 83, in get_top_level_apis
    top_level_apis = self.__filter_top_level_apis__(root_paths)
  File "/home/timur/venvs/freestone/lib/python3.6/site-packages/rest_framework_swagger/urlparser.py", line 92, in __filter_top_level_apis__
    base_path = self.__get_base_path__(root_paths)
  File "/home/timur/venvs/freestone/lib/python3.6/site-packages/rest_framework_swagger/urlparser.py", line 100, in __get_base_path__
    base_path = os.path.commonprefix(root_paths)
  File "/home/timur/venvs/freestone/lib/python3.6/genericpath.py", line 76, in commonprefix
    if not isinstance(m[0], (list, tuple)):
TypeError: 'set' object does not support indexing
```

**Comment by timur-orudzhov** [comment]
It's not big deal to fix that - https://github.com/marcgibbons/django-rest-swagger/pull/720

**Comment by tuffnatty** [comment]
I've merged that to [my fork of the stable-0.3.x branch](https://github.com/tuffnatty/django-rest-swagger/tree/my-0.3.x) and I suggest you to check it out as it has some more important fixes (all pull requests are here as well, but the project maintainer is long-lost).
